beacon_node/beacon_chain/src/data_column_verification.rs

```diff
@@ -181,6 +185,16 @@ pub struct GossipVerifiedDataColumn<T: BeaconChainTypes, O: ObservationStrategy
     _phantom: PhantomData<O>,
 }
 
+impl<T: BeaconChainTypes, O: ObservationStrategy> Clone for GossipVerifiedDataColumn<T, O> {
+    fn clone(&self) -> Self {
+        Self {
+            block_root: self.block_root,
+            data_column: self.data_column.clone(),
```

Comment:
Why don't you add a new property to `GossipVerifiedDataColumn` named `should_import`?

Reply:
I don't feel like import logic belongs to gossip verification, so I'd prefer to avoid that and leave the handling logic at the caller site, rather than having import logic at multiple places.